docs/Admin.md

```diff
@@ -442,6 +443,17 @@ const Foo = () => (
 export default Foo;
 ```
 
+## `basename`
+
+If you need to host react-admin in a subdirectory on your web host, e.g.
+
+`http://example.com/admin/`
+
+please specify a [basename](https://reacttraining.com/react-router/web/api/BrowserRouter/basename-string) via the basename prop to teach react-router..
```

Comment:
Remove "please", start the line by a capital letter, put basename between backticks

By the way, this sentence doesn't make sense. "To teach react-router": to teach what? And readers probably don't know about react-router so far in the documentation.

Reply:
Thanks for the feedback! 
I'm still reading through existing docs to grok the style and this isn't finished yet. I'll incorporate your instructions